source/deployment/ldap-group-sync.rst

```diff
@@ -72,7 +79,7 @@ Adding Default Teams or Channels For The Group
 --------------------------------------------
 To add the teams and channels that you want the group members to default in, select either **Add Team** or **Add Channel** from the **Add Team or Channel** button.
 
-.. image:: ../images/Add_Team_Or_Channel.png
+.. image:: ../images/Add_Team_Channel_Role.png
```

Comment:
@wiersgallak, I've compared this to the existing screenshot and the icons don't match up. Wondering if I didn't set up my environment properly or they've changed? 

Existing screenshot is here: https://docs.mattermost.com/_images/Add_Team_Or_Channel.png.

Reply:
The UI has changed. New image below: 
![Add_Team_Or_Channel](https://user-images.githubusercontent.com/39744472/74055193-25c01c00-49a5-11ea-9ccf-084cf79c081e.png)